lib/python/compliance_checker_imos_files_config.json

```diff
@@ -1,47 +1,40 @@
+    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/DWM/DA/EAC0500/Velocity/IMOS_DWM-DA_AETVZ_20161025T000000Z_EAC0500_FV01_EAC0500-2018-WORKHORSE-ADCP-120_END-20161112T062229Z_C-20190218T230812Z.nc",
     "check_params": {
       "check_success_tests": [ "cf:1.6", "imos:1.4" ],
       "skip_checks": [ "check_dimension_order" ]
     }
   },
   "abos_sots_adcp": {
-    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/eMII/test_files/ABOS/SOTS/IMOS_ABOS-SOTS_AETVZ_20130426_SOFS_FV01_SOFS-4-2013-Aquadopp-Profiler-AQD11011-4.14-1.01m_END-20131028_C-20190124.nc",
+    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/DWM/SOTS/2013/IMOS_DWM-SOTS_AETVZ_20130426_SOFS_FV01_SOFS-4-2013-Aquadopp-Profiler-AQD11011-4.14-1.01m_END-20131028_C-20190124.nc",
     "check_params": {
       "check_success_tests": [ "cf:1.6", "imos:1.4" ],
       "skip_checks": [ "check_dimension_order", "check_convention_globals" ]
     }
   },
   "abos_sots_parflux": {
-    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/eMII/test_files/ABOS/SOTS/IMOS_ABOS-SOTS_KF_20180322_SAZ47_FV01_SAZ47-20-2018_PARFLUX-Mark78H-21-1000m_END-20180322_C-20200504.nc",
+    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/DWM/SOTS/2018/IMOS_DWM-SOTS_KF_20180322_SAZ47_FV01_SAZ47-20-2018-McLane-PARFLUX-Mark78H-21-1000m_END-20190225_C-20210125.nc",
     "check_params": {
       "check_success_tests": [ "cf:1.6", "imos:1.4" ],
       "skip_checks": [ "check_dimension_order", "check_convention_globals" ]
     }
   },
   "abos_sots_wave": {
-    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/eMII/test_files/ABOS/SOTS/IMOS_ABOS-SOTS_RW_20180306_SOFS_FV01_SOFS-7-2018-3DM-GX1-5357-0m_END-20180316_C-20190605_subset.nc",
-    "check_params": {
-      "check_success_tests": [ "cf:1.6", "imos:1.4" ],
-      "skip_checks": [ "check_dimension_order", "check_convention_globals" ]
-    }
-  },
-  "abos_sots_gridded": {
-    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/eMII/test_files/ABOS/SOTS/IMOS_ABOS-SOTS_REFOBGTPCS_20090922_Pulse_FV02_Pulse-6-2009-Gridded-Data_END-20100325_C-20190522.nc",
+    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/DWM/SOTS/2018/IMOS_DWM-SOTS_RW_20180306_SOFS_FV01_SOFS-7-2018-3DM-GX1-5357-0m_END-20180316_C-20190605.nc",
     "check_params": {
       "check_success_tests": [ "cf:1.6", "imos:1.4" ],
       "skip_checks": [ "check_dimension_order", "check_convention_globals" ]
     }
   },
   "abos_sots_rt": {
-    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/eMII/test_files/ABOS/SOTS/IMOS_ABOS-SOTS_W_20190318_SOFS_FV00_SOFS-8-2019-MRU-Surface-wave-height-realtime.nc",
+    "file_url": "https://s3-ap-southeast-2.amazonaws.com/imos-data/IMOS/DWM/SOTS/2019/real-time/IMOS_DWM-SOTS_W_20190318_SOFS_FV00_SOFS-8-2019-MRU-Surface-wave-height-realtime.nc",
```

Comment:
Comment might be less valid here (no creation date in filename), but who knows we might change the directory structure in the future and we don't want that to impact the unit test.